app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -463,6 +469,62 @@ class CtaViewModelTest {
         assertTrue(value !is DaxBubbleCta.DaxEndCta)
     }
 
+    @Test
+    fun whenRefreshCtaOnHomeTabAndConceptTestFeatureActiveAndIntroCtaWasNotPreviouslyShownThenCovidCtaNotShown() = runBlockingTest {
+        setConceptTestFeature()
+        whenever(mockDismissedCtaDao.exists(CtaId.DAX_INTRO)).thenReturn(false)
+        whenever(mockDismissedCtaDao.exists(CtaId.COVID)).thenReturn(false)
+
+        val value = testee.refreshCta(coroutineRule.testDispatcher, isBrowserShowing = false)
+        assertFalse(value is HomeTopPanelCta.CovidCta)
+        assertTrue(value is DaxBubbleCta.DaxIntroCta)
```

Comment:
This assert appears in the next text. I would remove it from here, as this condition doesn't appear in the test name either.

Reply:
Removed!